analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -246,34 +239,18 @@ open class Npm(
                         || hash == Hash.NONE || vcsFromPackage == VcsInfo.EMPTY
 
                 if (hasIncompleteData) {
-                    // Download package info from registry.npmjs.org.
-                    // TODO: check if unpkg.com can be used as a fallback in case npmjs.org is down.
-                    val encodedName = if (rawName.startsWith("@")) {
-                        "@${URLEncoder.encode(rawName.substringAfter('@'), "UTF-8")}"
-                    } else {
-                        rawName
-                    }
-
-                    log.debug { "Resolving the package info for '${id.toCoordinates()}' via NPM registry." }
-
-                    OkHttpClientHelper.downloadText("$npmRegistry/$encodedName/$version").onSuccess {
-                        val versionInfo = jsonMapper.readTree(it)
+                    val process = NpmCli.run("view", "--json", "$rawName@$version")
```

Comment:
This command can be limited to only the fields we require, you could add `, "description", "homepage", "dist"`. I was hoping this would make the command faster, but I could not see a significant difference locally.

Reply:
It's probably even more "expensive", as I guess the limiting is implemented by post-processing the whole data.

Reply:
It was never slower, but also not significantly faster. I guess printing the whole json to the console is slower than the filtering.

Reply:
Yeah, I was just referring to the fact that theoretically doing more work (the filtering) should take longer, but it's certainly below the measurable threshold.